accepted/00076-state-channels-visibility.md

```diff
@@ -0,0 +1,218 @@
+
+After:
+![System Details Highstate AFTER](images/state-channels-visibility/Systems_Details_States_Highstate_AFTER.png)
+
+### `Systems > Details > States > Configuration Channels`
+Before:
+![System Details BEFORE](images/state-channels-visibility/Systems_Details_States_ConfigurationChannels_BEFORE.png)
+
+After:
+![System Details AFTER 1](images/state-channels-visibility/Systems_Details_States_ConfigurationChannels_AFTER_1.png)
+![System Details AFTER 2](images/state-channels-visibility/Systems_Details_States_ConfigurationChannels_AFTER_2.png)
+
+### `Systems > System Groups > Details > States > Configuration Channels`
+This page represents the assignment connection between a system group and state channels. All the systems in this system group will inherit the selection set of channels here. This should be only a list of State Channels with the possibility of simply select/deselect channels and Apply changes. If there is a need to show only the channel selected, a filter to the list page should be added instead.
+
+Before:
+![System Groups BEFORE](images/state-channels-visibility/Systems_SystemGroups_Details_States_ConfigurationChannels_BEFORE.png)
+
+After:
+![System Groups AFTER](images/state-channels-visibility/Systems_SystemGroups_Details_States_ConfigurationChannels_AFTER.png)
+
+### `Systems > System Set Manager > Overview > Configuration > Subscribe to Channels`
+Before:
+![SSM Subscribe To Channels BEFORE](images/state-channels-visibility/Systems_SSM_Configuration_SubscribeToChannels_BEFORE.png)
+
+After:
+![SSM Subscribe To Channels AFTER](images/state-channels-visibility/Systems_SSM_Configuration_SubscribeToChannels_AFTER.png)
+
+### `Systems > System Set Manager > Overview > Configuration > Unsubscribe to Channels`
+Before:
+![SSM Unsubscribe From Channels BEFORE](images/state-channels-visibility/Systems_SSM_Configuration_UnsubscribeFromChannels_BEFORE.png)
+
+After:
+![SSM Unsubscribe From Channels AFTER](images/state-channels-visibility/Systems_SSM_Configuration_UnsubscribeFromChannels_AFTER.png)
+
+### `Configuration > Overview`
+Before:
+![Configuration Overview BEFORE](images/state-channels-visibility/Configuration_Overview_BEFORE.png)
+
+After:
+![Configuration Overview AFTER](images/state-channels-visibility/Configuration_Overview_AFTER.png)
+
+### `Configuration > Channels`
+Before:
+![Configuration Channels BEFORE](images/state-channels-visibility/Configuration_Channels_BEFORE.png)
+
+After:
+![Configuration Channels AFTER](images/state-channels-visibility/Configuration_Channels_AFTER.png)
+
+### `Configuration > Channels > Details > Overview`
+Before:
+![Configuration Channels Detail Overview BEFORE](images/state-channels-visibility/Configuration_Channels_Details_Overview_BEFORE.png)
+
+After:
+![Configuration Channels Detail Overview AFTER](images/state-channels-visibility/Configuration_Channels_Details_Overview_AFTER.png)
+
+### `Configuration > Channels > Details > Systems > Inheriting Systems`
```

Comment:
```suggestion
### `Configuration > Channels > Details > Systems > Inherited Systems`
```

Please check with native English speakers if "inherited" is a good term here - we might want to prefer "indirectly assigned" or something similar. I just do not feel qualified enough to be authoritative.